packages/ckeditor5-adapter-ckfinder/docs/api/adapter-ckfinder.md

```diff
@@ -17,8 +17,10 @@ Check out the {@link features/image-upload comprehensive "Image upload" guide} t
 
 ## Installation
 
+This package is part of our open-source aggregate package
```

Comment:
Find and replace:

This package is part of our open-source aggregate package.

or

This package is part of our open-source aggregate package: